mmocr/utils/ocr.py

```diff
@@ -633,6 +749,10 @@ def det_recog_kie_inference(self, det_model, recog_model, kie_model=None):
 
     # Separate det/recog inference pipeline
     def single_inference(self, model, arrays, batch_mode, batch_size=0):
+        # todo: support batch mode for tesseract
+        if model == 'Tesseract':
+            return self.tesseract_det_inference(arrays)
```

Comment:
Hi Garven！
There is a bug here. No matter we are using recognition or detection API from tesserocr, the `model`'s name will always be  'Tesseract' as defined at `line276` and `line328`. So it just call the detect function here. I suppose we can modify like this? 
```python
# at line 373
self.detect_model = self.td -> self.detect_model = 'Tesseract_det'
...
# at line 389
self.recog_model = self.tr -> self.recog_model = 'Tesseract_recog'
...
# at line 753
if model == 'Tesseract_det':
   return self.tesseract_det_inference(arrays)
elif model == 'Tesseract_recog':
   return self.tesseract_recog_inference(arrays)
```

Reply:
Thanks, I think it will works